### DISCUSSION TITLE
화이트모드일때 코드블럭 하이라이트가 사라지는 현상

### DISCUSSION THREAD

**Discussion by JungJaeLee-JJ** [pronist/hELLO#83] [Q&A] [answered]
기존 closed 된 이슈에서 이와 같은 현상이 있었는데, 저 또한 화이트모드일 때 코드 블럭에 하이라이트가 사라지는 현상을 겪고있습니다.

이러한 문제도 있고, 다크모드가 더 이쁜 것 같아서 다크모드만 사용하고 싶은데, css 를 어떻게 수정하면 될까요?

**Comment by pronist** [comment]
@JungJaeLee-JJ 안녕하세요. 해당 현상이 발생하는 포스트 주소를 알 수 있을까요?

기본테마를 다크모드로 하시려면 스킨옵션에서 테마를 수동으로 설정하시고 테마변경 버튼을 off 시키시기 바랍니다.

**Comment by seovalue** [comment]
안녕하세요, 저는 다크모드일 때 코드가 검정색과 유사한 색으로 바뀌어 보이지 않는 문제가 발생하는데 혹시 어떻게 해결하는 지 알려주실 수 있으실까요? 블로그 링크도 첨부합니다!

https://joanne.tistory.com/

**Comment by pronist** [comment]
@seovalue 그 버그는 **3.4.0** 버전에서 처리된바 있습니다. [패치노트](https://github.com/tistory-project/hELLO/blob/master/CHANGELOG.md) 참고해주세요.

**Comment by JungJaeLee-JJ** [comment]
블로그 주소 - https://blog.jjlovechoco.com/40 
다크모드에서 화이트모드 변경시, 코드의 색이 바뀌지 않는 것 같습니다.

**Comment by pronist** [comment, marked as answer]
@JungJaeLee-JJ 버그라기에는, 스킨에 제가 넣지 않은 코드가 들어가있습니다. 어째서 **https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/monokai.min.css** 에 해당하는 테마가 블로그에 들어가 있으신걸까요? 제가 생각하기에는 티스토리 플러그인 중에 *코드 하이라이팅*이 활성화 되어있는게 아닌지 의심됩니다. 스타일이 겹쳐있기 때문에 덮어쓰기 되어 색상변경이 안 된 것이라고 생각됩니다.

티스토리 플러그인 중에, 코드 하이라이팅과 관련된 플러그인을 한 번 비활성화 해보시겠어요? 그렇지 않고서야 tistory 에서 자동으로 스크립트를 구입하는 부분에 highlight.js 가 포함되어 있을리가 없으니까요 ..

**Comment by JungJaeLee-JJ** [comment]
code 하이라이트 관련 플러그인을 끄니깐 정상 작동 됩니다. 감사합니다.

**Comment by pronist** [comment]
@JungJaeLee-JJ 그나저나 이 문제는 마주하는 분들이 꽤나 많은 것 같네요. 스킨 소개글에 트러블 슈팅쪽에도 남겨놔야 될 것같습니다. 리포팅해주셔서 고맙습니다 :)

**Reply by jinny-l** [reply]
안녕하세요. 좋은 스킨 제작해주셔서 감사합니다.  :)
저도 해당 문제를 겪다 이슈까지 찾아오게 되었는데요, 블로그 개설 시점인지 스킨 적용 시점인지는 모르겠지만
코드 하이라이팅 플러그인이 디폴트로 설정되어 있어 그런 것 같습니다!

혹시나 도움이 되실 것 같아 말씀드립니다.
